user-guide/Basic_Functionality/Documents/Managing_the_document_folders.md

```diff
@@ -16,11 +18,14 @@ The tree view on the left side of the Documents module contains several root fol
 
 Within each folder, you can **add, rename, or delete custom folders** through the right-click menu.
 
+> [!NOTE]
+> Adding subfolders on element level is only supported from DataMiner 10.4.0 [CU5]/10.4.8 onwards.<!-- RN 39076+39876 -->
+
 ## The Elements folder
 
 <!-- RN 21997 -->
 
-On the *Documents* page of an element, there is also a default *Elements* folder containing a subfolder for that specific element. This subfolder contains documents that are associated with that particular element only. It is not possible to add or rename such element folders.
+On the *Documents* page of an element, there is a default *Elements* folder containing a subfolder for that specific element. This subfolder contains documents that are associated with that particular element only. It is not possible to add or rename such element folders.
 
 The documents in such an element folder are stored on the DataMiner Agent hosting the element, and they are not synced with other Agents in the cluster. This makes this folder especially useful for storing larger files. However, note that these documents cannot be opened through Cube.
```

Comment:
I would mention the improved download support as follows:
> However, note that these documents ~_cannot be opened through Cube_~ can only be saved when Cube is connected to the agent where that element is hosted.

Note that there is a difference between 'open' and 'save' as explained in the note at the bottom of Consulting_documents_hyperlinks_or_email_addresses.md:
* open = launch the system browser to https://dma/Documents/.../myfile.txt = still not supported
* save = download the file with Cube via SLNet messages